app/services/proofing/lexis_nexis/ddp/proofer.rb

```diff
@@ -14,7 +14,9 @@ class Proofer < LexisNexis::Proofer
                             :address1,
                             :city,
                             :state,
-                            :zipcode
+                            :zipcode,
+                            :input_ip_address,
```

Comment:
`input_ip_address` makes it sound like the user has input it directly? what if on our side we call it `request_ip` or something?

Reply:
Also can we add the code that wires in the actual user's IP to send to LN? I think that would have to plumb through the `ResolutionProofingJob`

Reply:
I want to confirm we want to update this and anything dependent on it: 
https://github.com/18F/identity-idp/blob/f68c8f042480fe02eb03665f567bc6b5e432295a/app/jobs/resolution_proofing_job.rb#L15

Reply:
yes and we need to make sure new arguments added are nil-safe (backward compatibility during a deploy)